src/main/java/trackitnus/model/lesson/LessonComparator.java

```diff
@@ -6,15 +6,15 @@
     /**
      * Compares 2 lessons in chronological order to facilitate sorting.
      *
-     * @param firstLesson First lesson to compare.
+     * @param firstLesson  First lesson to compare.
```

Comment:
Is the extra space supposed to be there?

Reply:
No 😢 I think this happened during merging

Reply:
Ah yes, it's the formatter of IntelliJ, it auto-align the comments of params